hail_layout.ld

```diff
@@ -0,0 +1,11 @@
+/* Layout for the nRF52-DK, used by the examples in this repository. */
```

Comment:
Should "nRF52-DK" be replaced with "Hail"?

Reply:
Right.